docs/ENVIRONMENT.md

```diff
@@ -164,6 +164,10 @@ All tools:
 * `$SYSTEMD_NSPAWN_TMPFS_TMP=0` — if set, do not overmount `/tmp/` in the
   container with a tmpfs, but leave the directory from the image in place.
 
+* `$SYSTEMD_NSPAWN_CHECK_OS_RELEASE=0` — if set, do not fail when trying to
+  boot an OS tree without an os-release file (useful when trying to boot a
+  container with empty /etc and bind-mounted /usr)
```

Comment:
please enclose `/etc` and `/usr` in backticks

oh, and we generally try to suffix paths that reference dirs in our docs with a `/`. Hence please change /etc → /etc/ and /usr → /usr/